zebra-chain/src/parallel/tree.rs

```diff
@@ -11,7 +11,7 @@ use crate::{
 };
 
 /// An argument wrapper struct for note commitment trees.
-#[derive(Clone, Debug)]
+#[derive(Clone, Debug, Default)]
```

Comment:
Let's document that the default represents the genesis treestate. (Which is how other tree defaults work elsewhere in production code.)

We usually derive a bunch of standard traits, not sure why that didn't happen here.

```suggestion
#[derive(Clone, Debug, Default, Eq, PartialEq)]
```

Reply:
> Let's document that the default represents the genesis treestate. (Which is how other tree defaults work elsewhere in production code.)

Done in [71fbc1d](https://github.com/ZcashFoundation/zebra/pull/7923/commits/71fbc1d3d38c5a79e8f7322660b32d9185d0826e). I also documented the difference between the default root on its own, and the default root of the default tree in [2087424](https://github.com/ZcashFoundation/zebra/pull/7923/commits/2087424df095a2f928f8597cceddeea3fd800e0e).